otelib/backends/services/dataresource.py

```diff
@@ -1,15 +1,26 @@
 """Common strategy for Download, Prase and Resource strategies."""
+from typing import TYPE_CHECKING
+
 import requests
 from oteapi.models import ResourceConfig
 
 from otelib.exceptions import ApiError
-from otelib.strategies.abc import AbstractStrategy
 
+from .base import AbstractServicesStrategy
```

Comment:
```suggestion
from otelib.backends.services.base import BaseServicesStrategy
```
This change should be the same for all other strategies.